subworkflows/local/arg.nf

```diff
@@ -142,9 +150,12 @@ workflow ARG {
         // Reporting
         // Note: currently hardcoding versions as unreported by DeepARG
         // Make sure to update on version bump.
-        HAMRONIZATION_DEEPARG ( DEEPARG_PREDICT.out.arg.mix( DEEPARG_PREDICT.out.potential_arg ), 'json', '1.0.2', params.arg_deeparg_db_version )
-        ch_versions = ch_versions.mix( HAMRONIZATION_DEEPARG.out.versions )
-        ch_input_to_hamronization_summarize = ch_input_to_hamronization_summarize.mix( HAMRONIZATION_DEEPARG.out.json )
+        HAMRONIZATION_DEEPARG_TSV ( DEEPARG_PREDICT.out.arg.mix( DEEPARG_PREDICT.out.potential_arg ), 'tsv', '1.0.2', params.arg_deeparg_db_version )
```

Comment:
I don't like adding too much channel manipulation within a module invocation (I find it much harder to read), could you instead move to something like

```suggestion
        ch_input_to_hamronizationdeeparg = DEEPARG_PREDICT.out.arg.mix( DEEPARG_PREDICT.out.potential_arg )
        HAMRONIZATION_DEEPARG_TSV ( ch_input_to_hamronizationdeeparg, 'tsv', '1.0.2', params.arg_deeparg_db_version )
```

I know you didn't add this, but I miust have missed it first time around

Reply:
And same for all other modules